packages/paypal-commerce-integration/src/CountryData.ts

```diff
@@ -0,0 +1,1222 @@
+export interface CountryData {
```

Comment:
Shall we move this to payment-integration-package? Since this is something we might need to other providers too, so we can save duplication later.